src/pages/ReviewEdit.js

```diff
@@ -1,9 +1,107 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+import { Form, FormGroup, Input, Label, Button } from 'reactstrap'
+
+const ReviewEdit = ({ updateReview, current_user, beers, reviews }) => {
+  const navigate = useNavigate()
+  let { id } = useParams()
+
+  const currentReview = reviews?.find((review) => review.id === +id)
+
+
+  useEffect(() => {
+    if (currentReview) {
+      setEditReview({
+        city: currentReview.city,
+        state: currentReview.state,
+        review_text: currentReview.review_text,
+        rating: currentReview.rating,
+        user_id: current_user.id,
+        beer_id: currentReview.beer_id,
+      })
+    }
+  }, [currentReview])
+
+  const [editReview, setEditReview] = useState({
+    city: '',
+    state: '',
+    review_text: '',
+    rating: 0,
+    user_id: current_user.id,
+    beer_id: currentReview ? currentReview.beer_id : 0,
```

Comment:
Is this association optional?  If so, this makes sense, if not, hardcoding 0 as the else will likely cause problems in the future.